lib/core/numbers.toit

```diff
@@ -564,6 +564,33 @@ abstract class int extends num:
   */
   static MIN ::= -MAX - 1
 
+  /** The minimum signed 8-bit integer value. */
+  static MIN-8 ::= -128
+  /** The maximum signed 8-bit integer value. */
+  static MAX-8 ::= 127
+  /** The minimum signed 16-bit integer value. */
+  static MIN-16 ::= -32_768
+  /** The maximum signed 16-bit integer value. */
+  static MAX-16 ::= 32_767
+  /** The minimum signed 24-bit integer value. */
+  static MIN-24 ::= -8_388_606
```

Comment:
This should be -8_388_608

Reply:
done. switched to hex and "-max - 1" pattern.